chingu-coders/Voyage2-Turtles-02
# added some advanced pomodoro functionality

## Purpose
* To add more options to pomodoro clock

## Approach/Fixes
* Implements advanced features like ability to pause, reset and toggle periods of pomodoro clock

## Testing
* Check out the pull request
* click the "start pomodoro cycle" button
* confirm that you see buttons for stop, pause, reset, and break
*click pause button and confirm that cycle pauses. click again to confirm that cycle continues
* click reset and confirm that time goes back to 1:00
* click break. Confirm that cycle switches to break, and button text switches to "work"
* click stop and confirm that pomodoro cycle ends and regular clock is shown with "start pmodoro cycle" button
